60_networks.Rmd

```diff
@@ -0,0 +1,664 @@
+from Student's t-test in the lower triangle). 
+See `?calcGCM` and `?testGCM` for details.
+
+```{r netAnalyze_single, fig.width=7, fig.height=7, out.width='75%'}
+peerj_netprops <- netAnalyze(peerj_net, 
+                             centrLCC = TRUE,
+                             clustMethod = "cluster_fast_greedy",
+                             hubPar = "eigenvector",
+                             hubQuant = 0.95,
+                             weightDeg = FALSE, 
+                             normDeg = FALSE)
+```
+
+```{r summary_single}
+summary(peerj_netprops)
+```
+
+Interpretation of some of the results:
+
+* The largest connected component (LCC) has 35 nodes and the network contains 6 singletons.
+* 10 clusters have been identified, containing two to eight nodes.
+* The detected hub nodes are Citrobacter, Escherichia, and Salmonella, which by definition are the nodes with the highest eigenvector centrality.
+* These three genera do also have a high degree and closeness centrality, but a lower betweenness centrality.
+* The average path length is 3.56. This means that on average it takes 3.56 steps (step length is the average dissimilarity) to get from one node to another node in the network.
+* Low values of edge density and the connectivity measures indicate that the network is rather sparse and not robust to perturbations (i.e., removal of nodes or edges).
+
+### Network plot
+
+Further insight into the network structure can be gained by visualizing the network, which is done using NetCoMi's plot function. The help page can be accessed via `?plot.microNetProps`.
+
+#### Highlight node properties
+
+In the first plot, node colors and sizes are used to visualize certain network properties.
+
+Here is a short explanation of our argument configuration:
+
+* `layout = "spring"`: A force-directed layout is used, placing strongly connected nodes close together and nodes with less edge weight farther apart.
+* `repulsion = 0.98`: Default is 1. By decreasing the repulsion, nodes are placed farther apart.
+* `rmSingles = TRUE`: Singletons are removed.
+* `labelScale = FALSE`: Labels are not scaled by node size.
+* `nodeSize = "eigenvector"`: Nodes are scaled by eigenvector centrality.
+* `nodeSizeSpread = 3`: Node sizes are more similar if the value is decreased. This argument (in combination with cexNodes) is useful to enlarge small nodes while keeping the size of big nodes. Default is 4.
+* `nodeColor = "cluster"`: Node colors represent clusters.
+* `edgeTranspHigh = 20`: Edges with high edge weight are given some transparency so that node labels overlapping the edges are still readable.
+* `cex...`: Scaling of the node sizes, labels, title, etc.
+
+**Note:** Nodes are sometimes placed too close together so that the labels overlap. You may need to play around with the repulsion until you find a value where the labels are legible, but also the clusters are still well recognizable.
```

Comment:
Again, consider citing function documentation in the package instead of replicating argument descriptions here.